chapters/02-estrutura-do-programa.md

```diff
@@ -87,30 +87,26 @@ A coleção de variáveis e seus valores que existem em um determinado tempo é
 
 ## Funções
 
-Muitos dos valores fornecidos no ambiente padrão são do tipo *function* (função). Uma função é um pedaço de programa envolvido por um valor. Este valor pode ser aplicado, a fim de executar o programa envolvido. Por exemplo, no ambiente do navegador, a variável `alert` detém uma função que mostra uma pequena caixa de diálogo com uma mensagem. É usada assim:
+Muitos dos valores fornecidos no ambiente padrão são do tipo `function` (função). Uma função é um pedaço de programa envolvido por um valor. Este valor pode ser aplicado a fim de executar o programa envolvido. Por exemplo, no ambiente do navegador, a variável `alert` detém uma função que mostra uma pequena caixa de diálogo com uma mensagem. É usada da seguinte forma:
 
 ```javascript
-
 alert("Bom dia!");
-
 ```
 
 [JSFiddle](http://jsfiddle.net/K3Fe3/)
 
-Executar uma função é denominado *invocando*, *chamando* ou *aplicando* uma função. A notação para fazer isso é colocar um parênteses depois de uma expressão que produza um valor de uma função. Normalmente você vai referenciar diretamente a uma variável que detém uma função. Os valores entre os parênteses são dados ao programa dentro da função. No exemplo, a função `alert` usou a string que foi dada como o texto para ser mostrado na caixa de diálogo. Valores dados a funções são chamados *argumentos*. A função `alert` precisa somente de um, mas outras funções podem precisar de diferentes quantidades ou tipos de argumentos.
+Executar uma função é denominado *invocar*, *chamar* ou *aplicar* uma função. A notação para fazer isso é colocar um parênteses depois de uma expressão que produza um valor de uma função. Normalmente você vai referenciar diretamente uma variável que detenha uma função. Os valores entre os parênteses são dados ao programa dentro da função. No exemplo, a função `alert` usou a string que foi passada como o texto a ser mostrado na caixa de diálogo. Os valores passados para funções são chamados de *argumentos*. A função `alert` precisa somente de um deles, mas outras funções podem precisar de diferentes quantidades ou tipos de argumentos.
```

Comment:
- Executar uma função é denominado *invocar*, *chamar* ou *aplicar* uma função [...] > Execução de uma função é chamada invocando, chamando ou aplicando [...]

---

Original: "You can call a function by putting parentheses after an expression that produces a function value"
Tradução: "Você pode chamar uma função colocando os parênteses depois da expressão que produz um valor de função."

---

[...] "Normalmente você vai referenciar diretamente uma variável que detenha uma função." [...] > "Normalmente você irá usar diretamente o nome da variável que contém uma função"

---

[...] "são dados ao programa dentro da função." [...] > [...] são enviados para o programa dentro da função. [...]

---

string > \`string`

---

\*argumentos\* > argumentos